### PR TITLE
 Add GHA build, rename scope to @evidenceprime 

### PR DESCRIPTION
**Pull request by przemekrEP**

### CODE REVIEW

File: .github/workflows/cicd.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        with:
+          node-version: 20
+          cache: 'yarn'
+          registry-url: https://npm.pkg.github.com
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build
+        run: yarn build
```
Comment (salceson) [inline .github/workflows/cicd.yaml:31]: Can we add running tests before build + reporting results? If this is not possible, then we can remove permission for `checks` and `statuses`.

File: dist/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@ep/buffer",
+  "name": "@evidenceprime/buffer",
```
Comment (salceson) [inline dist/package.json:2]: This is about pipelines def: if we are removing this, let's remove pipeline on Azure side.

**Review by salceson — APPROVED** [review]
I am approving, the following comments are nice to have, not necessary to change.

File: .github/workflows/cicd.yaml (hunk 1 above)
Comment (przemekrEP) [reply .github/workflows/cicd.yaml:31]: Looks like tests here require credentials to some SaaS (or a local browser manually launched) to run.

**Pull request merged**